docs/java-rest/high-level/apis/indices_aliases.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[java-rest-high-indices-aliases]]
+=== Index Aliases API
+
+[[java-rest-high-indices-aliases-request]]
+==== Index Aliases Request
+
+The index aliases API allows aliasing an index with a name, with all APIs 
+automatically converting the alias name to the actual index name.
+
+An `IndicesAliasesRequest` must have at least one `AliasActions`:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[indices-aliases-request]
+--------------------------------------------------
+<1> Creates an `IndicesAliasesRequest`
+<2> Creates an `AliasActions` that aliases index `test1` with `alias1`
+<3> Add the alias action to the request
+
+The following action types are supported: `add` - alias an index, `remove` - 
+remove the alias assosiated with the index and `remove_index` - deletes the 
+index. 
+
+Defining an alias with filters provide an easy way to create different 
+"views" of the same index. The filter can be defined using Query DSL and is 
+applied to all Search, Count, Delete By Query and More Like This operations 
+with this alias.
+
+It is possible to associate routing values with aliases. This feature can be 
+used together with filtering aliases in order to avoid unnecessary shard 
+operations.
```

Comment:
I wonder if we should explain here the feature/API itself or rather just mention how to use it and point to the API docs to know more about  it. What do you think about rephrasing to just say e.g. "use this to set a routing value" etc. without explaining what routing is in this page?

Reply:
Do you mean to follow the low level client docs approach? Just say how to use requests / responses with a few examples. 
Or do you mean to have a simple example for every supported API with a request / response and a link to the API docs?

Reply:
besides the link, I wouldn't explain here what the API does and how the different options work. Just how to use the options with the client. At least I think this is what we've done until now. Makes sense?